wave-3-game.rb

```diff
@@ -0,0 +1,130 @@
+require_relative 'lib/scrabble'
+
+module Scrabble
+  class Game
+    def initialize
+      @words = []
+      @players = setup_players
+      @tilebag = Tilebag.new
+
+    end
+
+    def play
+      start
+
+      while continue?
+        @players.each do |player|
+          puts "It is #{player.name}'s turn"
+
+          player.draw_tiles @tilebag
```

Comment:
Yes!
One of the things that we've had missing from our Wave 3 requirements is that when a word is played on the Player - that it actually validates that the word can be played with the given set of the letters, and that it adjusts the list of letters that the player has. Its been more of a soft requirements since this will generally break any/all tests they have until this point. 

Maybe we should add this as an explicit optional - and have this `draw_tiles` call commented out initially?

Reply:
I'll make a comment that it is an optional requirement that a player playing a word should be checked against their tile inventory.

I think that players can still draw tiles and have a tile inventory (and not comment this line out), it just might not affect how `play(word)` will act :P Is that confusing for them, though?